.github/workflows/ci.yml

```diff
@@ -39,8 +39,11 @@ jobs:
      - name: Run linter
        run: npx elm-format --validate src/ showcase/
 
+     - name: Run docs
+       run: npx elm-format --validate src/ showcase/
+
      - name: Run analyse
-       run: npx elm-analyse
```

Comment:
why are you removing the analyse?

Reply:
maybe the github diff is off but I see it this way:
![image](https://user-images.githubusercontent.com/469455/100911076-0dd6fd00-34cf-11eb-80fe-09b2cf3768ba.png)

Reply:
My mistake, I edited the wrong line.